### PR TITLE
Add Wasmer icon

### LINKED ISSUE #7643: Request: Wasmer Icon
### Brand Name

Wasmer

### Website

https://wasmer.io/

### Popularity Metric

Similarweb rank: 291,242 (https://www.similarweb.com/website/wasmer.io)
Github Stars: 12,700 (https://github.com/wasmerio/wasmer/)

### Official Resources for Icon and Color

Wasmer Full logo:
svg: https://wasmer.io/images/logo.svg
icon: https://wasmer.io/images/icon.svg

Wasmer icon

color: `#4946dd`

### Additional Comments

Wasmer has a large and active community. The project will only continue to grow. Thank you!!!!

### Contributing

Yes

### PR DESCRIPTION
**Pull request by syrusakbary**
image

**Issue:** closes #7643

**Similarweb rank:** 291,242 (https://www.similarweb.com/website/wasmer.io)
**Github Stars:** 12,700 (https://github.com/wasmerio/wasmer/)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

  - Why did you pick the hex value? Is the brand color
  - Did you manually vectorize the logo? No

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11764,6 +11764,12 @@
             "source": "https://github.com/wasmCloud/branding/",
             "guidelines": "https://github.com/wasmCloud/branding/"
         },
+        {
+            "title": "wasmer",
+            "hex": "4946DD",
+            "source": "https://github.com/wasmerio/wasmer.io/blob/0d425f5b4ace56496e75278e304f54492c46adde/public/images/icon.svg",
+            "guidelines": "https://wasmer.io/"
```
Comment (sisp) [inline _data/simple-icons.json:11771]: I can't find the logo guidelines on this website. Unless I've overlooked it somewhere, please delete this field.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11764,6 +11764,12 @@
             "source": "https://github.com/wasmCloud/branding/",
             "guidelines": "https://github.com/wasmCloud/branding/"
         },
+        {
+            "title": "wasmer",
```
Comment (sisp) [inline _data/simple-icons.json:11768]: I think it's spelled with a capital W.

```suggestion
            "title": "Wasmer",
```

File: icons/wasmer.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>wasmer</title><path d="M18.111 3.537c-.011.822-.5 1.208-1.111.86-.611-.353-1.106-1.307-1.111-2.146L12 0v4.651l5.561 3.222.55.32v7.763L22 18.206V5.794l-3.889-2.256Zm-5 3.034c-.011.822-.5 1.208-1.111.86-.611-.352-1.106-1.307-1.111-2.145l-3.89-2.252V7.41l5.562 3.222.55.32v8.038L17 21.241V8.828L13.11 6.57Zm-5 2.759c-.011.822-.5 1.208-1.111.86-.611-.353-1.106-1.307-1.111-2.146L2 5.794v12.413L12 24V11.586L8.111 9.33Z"/></svg>
```
Comment (sisp) [inline icons/wasmer.svg:1]: Capital W (see above):

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Wasmer</title><path d="M18.111 3.537c-.011.822-.5 1.208-1.111.86-.611-.353-1.106-1.307-1.111-2.146L12 0v4.651l5.561 3.222.55.32v7.763L22 18.206V5.794l-3.889-2.256Zm-5 3.034c-.011.822-.5 1.208-1.111.86-.611-.352-1.106-1.307-1.111-2.145l-3.89-2.252V7.41l5.562 3.222.55.32v8.038L17 21.241V8.828L13.11 6.57Zm-5 2.759c-.011.822-.5 1.208-1.111.86-.611-.353-1.106-1.307-1.111-2.146L2 5.794v12.413L12 24V11.586L8.111 9.33Z"/></svg>
```

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for your contribution, @syrusakbary! :pray:

The icon is perfectly identical with the source SVG image (almost identical data path even). :+1:

I have only a few minor requests.

File: _data/simple-icons.json [hunk 4]
```diff
@@ -11764,6 +11764,12 @@
...
+            "hex": "4946DD",
+            "source": "https://github.com/wasmerio/wasmer.io/blob/0d425f5b4ace56496e75278e304f54492c46adde/public/images/icon.svg",
+            "guidelines": "https://wasmer.io/"
+        },
```
Comment (syrusakbary) [inline _data/simple-icons.json:11772]:
```suggestion
        },
```

**Comment by syrusakbary** [comment]
I just fixed the suggestions @sisp . Thanks for reviewing and looking forward to having the PR merged!

**Comment by sisp** [comment]
@syrusakbary The tests are failing beause of the trailing comma after the source URL. When you remove it, they should pass.

**Comment by syrusakbary** [comment]
@sisip Thanks for following up :)
It should be fixed now!

**Review by sisp — APPROVED** [review]
LGTM now! :tada:

Thanks again for your contribution, @syrusakbary! :pray:

**Pull request merged**